README.md

```diff
@@ -23,23 +23,66 @@ auto main() -> int {
     return 42;
 }
 `
-let code_section = new CodeSection(code, 'c++')
-    // equivalent to   awesome_doc_code_sections.HTML_elements.CodeSection
-
+let code_section = new CodeSection(code, 'cpp')
+    // equivalent to:  awesome_doc_code_sections.HTML_elements.CodeSection
 // attach `code_section` to the DOM ...
 ```
 
-Which is equivalent to :
+Which is equivalent to the following plain HTML:
+
+- As `div`
 
-```html
-<div class='awesome-doc-code-sections_basic-code-section' language="cpp">
+    ```html
+    <div class='awesome-doc-code-sections_basic-code-section' language="cpp">

```

Comment:
If its a section why not use a <section> tag instead of the <div> one? (SEO purpose)

Reply:
Good point : this is not part of the documentation yet, but some `Markdown`-based tools forbid the use of custom HTML elements (and most elements others than `div`, `p`, and various common ones.